src/itensor.jl

```diff
@@ -1953,7 +1953,7 @@ end
 
 *(As::ITensor...; kwargs...)::ITensor = contract(As...; kwargs...)
 
-function contract!(C::ITensor, A::ITensor, B::ITensor, α::Number, β::Number=0)::ITensor
```

Comment:
This actually changes the meaning of the code (it changes the positional argument to a keyword argument) and is caused by a bug in an older version of JuliaFormatter: https://github.com/domluna/JuliaFormatter.jl/issues/533.

Could you revert this change and upgrade your version of JuliaFormatter and run it again? Note that just upgrading and running again will not fix it automatically, since it will see the new code with the semicolon as properly formatted code, even if from our perspective it is incorrect.

Reply:
That's a weird bug. Thanks for catching it! I updated JuliaFormatter, manually reverted this and another one below in a unit test, and reformatted.